kuma/core/tests/test_helpers.py

```diff
@@ -96,6 +97,9 @@ def test_message_with_url_is_link(self):
 class TestDateTimeFormat(UserTestCase):
     def setUp(self):
         super(TestDateTimeFormat, self).setUp()
+        self.default_timezone = timezone.get_default_timezone()
+        self.old_times = (datetime.fromordinal(733900)
```

Comment:
I don't mean to be obtuse, but I'm not a fan of using the `datetime.fromordinal` function here, even if it was used before. To quote the docs: `Return the datetime corresponding to the proleptic Gregorian ordinal, where January 1 of year 1 has ordinal 1.` Can't we just use a specific time and be done with it?